src/app/ui/utils/truncate-middle.ts

```diff
@@ -0,0 +1,23 @@
+function truncateHex(hex: string, offset = 5): string {
```

Comment:
I just moved this here so we could have the docs in `README` also. I can move it back if we need to

Reply:
Seems smart to me to move ui utilities here. 👍